README.md

```diff
@@ -230,19 +230,20 @@ Within Xcode head to the build phase - `Bundle React Native code and images`.
     ```jsx
     npm i && npm run pod-install
     ```
-7. Depending on the platform you are targeting, run your Android/iOS app in production mode.
-8. Upon completion, the generated source map can be found at:
+4. Depending on the platform you are targeting, run your Android/iOS app in production mode.
+5. Upon completion, the generated source map can be found at:
   Android: `android/app/build/generated/sourcemaps/react/productionRelease/index.android.bundle.map`
   IOS: `main.jsbundle.map`
+  web: `dist/merged-source-map.js.map`
```

Comment:
Noob question: Why haven't we added the desktop/dist/www/merged-source-map.js.map path here for desktop?

Reply:
@jayeshmangwani I think because the code for web and desktop is identical. At least when I was trying to profile desktop app using web source map everything was well, i. e. I could see all stack-traces correctly